x/bank/keeper/keeper.go

```diff
@@ -380,14 +380,16 @@ func (k BaseKeeper) MintCoins(ctx context.Context, moduleName string, amounts sd
 
 // BurnCoins burns coins deletes coins from the balance of the module account.
 // It will panic if the module account does not exist or is unauthorized.
-func (k BaseKeeper) BurnCoins(ctx context.Context, moduleName string, amounts sdk.Coins) error {
-	acc := k.ak.GetModuleAccount(ctx, moduleName)
+func (k BaseKeeper) BurnCoins(ctx context.Context, address []byte, amounts sdk.Coins) error {
```

Comment:
I mean the changelog entry about this.

Reply:
There is already one

Reply:
Sorry, I thought that was added under the `Improvements`